libnl3.yaml

```diff
@@ -0,0 +1,62 @@
+package:
+  name: libnl3
+  version: 3.7.0
+  epoch: 0
+  description: Library for applications dealing with netlink sockets
+  target-architecture:
+    - all
+  copyright:
+    - paths:
+        - "*"
+      attestation: TODO
+      license: LGPL-2.1-or-later
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - automake
+      - autoconf
+      - linux-headers
+      - flex
+      - bison
+      - libtool
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: 9fe43ccbeeea72c653bdcf8c93332583135cda46a79507bfd0a483bb57f65939
+      uri: https://github.com/thom311/libnl/releases/download/libnl3_7_0/libnl-${{package.version}}.tar.gz
+  - uses: patch
+    with:
+      patches: libnl3-musl.patch

```

Comment:
Do we need a musl patch if we use glibc rather than musl in wolfi?

Reply:
You're right.